pkgs/development/tools/omnisharp-roslyn/default.nix

```diff
@@ -1,15 +1,15 @@
 { buildDotnetModule
```

Comment:
I can see how the changes in this package are useful for testing the source build but maybe they should be in a separate PR?